examples/minimal/packages/client-react-external-wallet/src/mud/supportedChains.ts

```diff
@@ -1,3 +1,12 @@
 import { type MUDChain, mudFoundry } from "@latticexyz/common/chains";
 
-export const supportedChains: MUDChain[] = [mudFoundry];
+export const supportedChains: MUDChain[] = [
+  {
+    ...mudFoundry,
+    // This is not an actual explorer URL; however, Wagmi requires a URL here. Wagmi uses the `blockExplorers` config of this chain to determine the `blockExplorerUrls` value for `wallet_addEthereumChain`. The `blockExplorerUrls` must be either null or an array containing at least one URL; however, Wagmi exclusively employs the array format. Thus, we need to include at least one URL here.
```

Comment:
can we pull this out and fix it in the common package instead of here?

Reply:
will do! 💪